MathLib/ODE/CVodeSolver.h

```diff
@@ -12,7 +12,7 @@
 
 #include "BaseLib/ConfigTree.h"
 
-#include "declarations.h"
+#include "OdeSolver-fwd.h"
```

Comment:
I think the new name is a bit misleading: You don't fwd-declare `ODESolver`.
When compared to `MathLib/ODESolver/Types.h`, I would use `Types.h` also here.